docs/HYPERVISORS.md

```diff
@@ -0,0 +1,85 @@
+# EVE's use of hypervisors and hardware assisted virtualization
+
+EVE is slightly unusual in how it looks at hypervisors and hardware assisted virtualization technology. Unlike some of the other systems that were clearly built as virtualization platforms (ESXi, Hyper-V, etc.) EVE's ultimate task is to be able to run [Edge Containers](ECOS.md) as efficiently (and securely!) as possible. A running ECO is represented as a series of tasks consuming resources provided by each Edge Node (such as CPU, RAM, volumes, network instances, etc.). Tasks can further improve their performance (and security) by requesting accelerator and/or isolation services typically provided by type-1 or type-2 hypervisors and backed by hardware assisted virtualization.
+
+This mapping of Tasks to resources creates an interesting architecture where things like virtualization capabilities can be consumed transparently without the user explicitly tweaking the virtualization knobs like it would be the case with systems like libvirt/virsh.
```

Comment:
> without the user explicitly tweaking the virtualization knobs like it would be the case with systems like libvirt/virsh

Simplify: "without the user needing to understand or tweak low-level virtualization knobs." No need to compare to others; rather emphasize the benefit.

Reply:
I'll leave it if it is OK with you -- since this happens to be our #1 FAQ item so far: "how come you guys are not using libvirt"